model/DataObject.php

```diff
@@ -1717,7 +1717,8 @@ public function getRemoteJoinField($component, $type = 'has_many', &$polymorphic
 	 * @todo Implement query-params
 	 */
 	public function getManyManyComponents($componentName, $filter = "", $sort = "", $join = "", $limit = "") {
```

Comment:
It looks like $join is ignored here.

I'd favour deprecating ALL the optional params and telling people to manipulate the list directly